doc/source/getting-started-mac-tutorial.rst

```diff
@@ -0,0 +1,155 @@
+macOS Tutorial
+==============
+
+This guide will give you a brief overview of how volatility3 works as well as a demonstration of several of the plugins available in the suite.
+
+Acquiring memory
+----------------
+
+Volatility3 does not provide the ability to acquire memory. The example below is an open source tool. Other commercial tools are also available.
+
+* `osxpmem <https://github.com/Velocidex/c-aff4/releases/download/3.2/osxpmem_3.2.zip>`_
```

Comment:
You should consider whether it is appropriate to link a specific version of the tool. 3.2 may be the latest version at the moment, but over time this could be a point to update every time.

Reply:
I agree, but I mentioned version 3.2 for a specific reason. The c-aff4 tool now includes versions 3.3 and 3.4, but these only support Windows and Linux. As far as I can tell, version 3.2 is the last one to support OSX. So, I intentionally referred to a particular version instead of the most recent one.